test/topicFilters.spec.ts

```diff
@@ -61,6 +83,57 @@ async function afterEachTest(): Promise<void> {
   await namespace.close();
 }
 
+async function sendOrders(): Promise<void> {
+  const data = [
+    { Color: "blue", Quantity: 5, Priority: "low" },
+    { Color: "red", Quantity: 10, Priority: "high" },
+    { Color: "yellow", Quantity: 5, Priority: "low" },
+    { Color: "blue", Quantity: 10, Priority: "low" },
+    { Color: "blue", Quantity: 5, Priority: "high" },
+    { Color: "blue", Quantity: 10, Priority: "low" },
+    { Color: "red", Quantity: 5, Priority: "low" },
+    { Color: "red", Quantity: 10, Priority: "low" },
+    { Color: "red", Quantity: 5, Priority: "low" },
+    { Color: "yellow", Quantity: 10, Priority: "high" },
+    { Color: "yellow", Quantity: 5, Priority: "low" },
+    { Color: "yellow", Quantity: 10, Priority: "low" }
+  ];
+
+  for (let index = 0; index < data.length; index++) {
+    const element = data[index];
+    const message: SendableMessageInfo = {
+      body: "",
+      messageId: generateUuid(),
+      correlationId: `${element.Priority}`,
+      label: `${element.Color}`,
+      userProperties: {
+        color: `${element.Color}`,
+        quantity: `${element.Quantity}`,
+        priority: `${element.Priority}`
+      }
+    };
+    await topicClient.send(message);
+  }
+}
+
+async function receiveOrders(client: SubscriptionClient): Promise<ServiceBusMessage[]> {
+  const receivedMsgs: ServiceBusMessage[] = [];
+  const receiveListener = client.receive(
+    (msg: ServiceBusMessage) => {
+      receivedMsgs.push(msg);
+      return Promise.resolve();
+    },
+    (err: Error) => {
+      should.not.exist(err);
+    }
+  );
+
+  await delay(3000);
```

Comment:
Lets update this to 5 seconds. Sometimes when I have run the streaming receiver, some messages never came in until I waited for 5 seconds